doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -474,18 +470,6 @@ The requirements listed in this section are optional, and are not required in or
 | `req.inf.acc.03` | Acceleration | The Architecture **should not** rely on SR-IOV PCI-Pass through to provide acceleration to VNFs. | |
 | `req.inf.img.01` | Image | The Architecture **should** make the immutable images available via location independent means. | [RA-1 4.3.1.2. "Glance"](./chapter04.md#4312-glance) |
 
-<!---
```

Comment:
Do not delete

Reply:
Should we uncomment it? Is there a plan to reintroduce this information in the published content?

Reply:
@EditKoselak The plan is to reintroduce but later

Reply:
Can you please re-introduce the commented requirements as they were deleted:
<!---
Commented out until RM defines requirements for SDN
| `req.inf.ntw.12` | Network | The SDN solution **should** be configurable via orchestration or VIM systems in an automated manner using openly published API definitions. | |
| `req.inf.ntw.13` | Network | The SDN solution **should** be able to support federated networks. | |
| `req.inf.ntw.14` | Network | The SDN solution **should** be able to be centrally administrated and configured. | |
--->